src/table/utils/styling-utils.js

```diff
@@ -65,6 +66,23 @@ export function getHeadStyle(layout, theme) {
 }
 
 export function getBodyStyle(layout, theme) {
```

Comment:
This function is very complex and not really self-explanatory. We need to look into if it can be simplified, or split up IMO

Reply:
Right, i gonna try to split it up

Reply:
Done. Any suggestion is welcome.